const erro = 0;

console.log(erro)

erro =1